pkg/cluster/nodes/create.go

```diff
@@ -130,6 +131,16 @@ func createNode(name, image, clusterLabel string, role config.NodeRole, extraArg
 		"--entrypoint=/usr/local/bin/entrypoint",
 	}
 
+	// pass proxy environment variables to be used by node's docker deamon
+	httpProxy := os.Getenv("HTTP_PROXY")
+	if httpProxy != "" {
+		runArgs = append(runArgs, "-e HTTP_PROXY=", httpProxy)
```

Comment:
shouldn't this be 
```suggestion
		runArgs = append(runArgs, "-e", "HTTP_PROXY="+httpProxy)
```
?

Reply:
Yes. Much clearer. Thanks.